### ISSUE TITLE
Creation of webhooks

### ISSUE THREAD

**Issue by michalisFr** [polkadot-js/phishing#1027]
We are trying to get several companies that specialise in scam detection and takedowns to integrate the phishing repo in their systems, firstly to pull domains from it and secondly to add any domains they find.

For the first part, they requested that webhooks are configured to notify them when new domains are added. This would be more effective than them constantly pulling data from it.

Is this something that can be done and, if so, how fast?

Thanks!

**Comment by jacogr** [comment]
No idea. Clueless as to how that works with GH actions (which is the only lever there is to pull)

**Comment by michalisFr** [comment]
So you are human after all! 😄

I'll have a look at GH documentation and see if I find anything. I mean if you can subscribe to repos it should be possible, right?

**Comment by jacogr** [comment]
You can certainly trigger stuff on commits - and the release commits have a standard form, eg. https://github.com/polkadot-js/phishing/commit/e316770aa4918ead195a1ffa37fc1621ae9c28f9

**Comment by michalisFr** [comment]
I found this:
https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks

Not sure what's easier: adding a rule as you suggest or setting up a webhook. Also, not sure if webhooks is something that you need to set up or whoever wants to get notified. Sorry, my knowledge on the subject is pretty limited.

**Comment by korn101** [comment]
Thanks team, we're eager to integrate to provide the most seamless service. Upon review, we've determined that traditional webhooks as we first proposed might be too much of a headache going forward and we would collectively prefer an approach using workflows. This will hopefully minimize the back-and-forth between our teams, and also allow us to bear most of the work ourselves. We will follow up with a PR, and an auth key that will need to be added to this repo's secrets.
